migrations/2018_09_22_004100_change_registration_tokens_columns.php

```diff
@@ -24,7 +24,7 @@
     },
 
     'down' => function (Builder $schema) {
-        $schema->table('auth_tokens', function (Blueprint $table) {
+        $schema->table('registration_tokens', function (Blueprint $table) {
```

Comment:
It was a mistake. I fixed this when I tried to make the TODO from #1674 work in this branch.

Since that turned out to be harder than expected, I postponed the TODO, but it still makes sense to get the fix in. Do you want me to extract this to `master`?

Reply:
Nope, I can imagine we merge this soon anyway. Just curious about why it got in in the first place.